build.rs

```diff
@@ -18,12 +18,14 @@ fn opencv_link() {
                 .filter_map(|entry| entry.ok())
                 .find(|entry| {
                     let file_name = entry.file_name().to_string_lossy().into_owned();
-                    file_name.starts_with("opencv_world") && !file_name.ends_with("d.lib")
+                    (file_name.starts_with("opencv_world") || file_name.starts_with("libopencv_world")) && !file_name.ends_with("d.lib")
                 });
             if let Some(opencv_world) = opencv_world_entry {
-                let opencv_world = opencv_world.path();
+                let opencv_world = opencv_world.file_name();
+                let opencv_world = opencv_world.into_string().unwrap();
+                let opencv_world_without_extension = opencv_world.trim_right_matches(|c: char| !c.is_numeric());
```

Comment:
`file_stem` is not working properly with "double-extensions" such as `.dll.a`. Because we know the shape of the exe `(lib?)opencw_world<some_numbers>.extension` we just trim everything after version number. We are also free to call `unwrap` after `into_string` because this shape cannot contain invalid UTF8 data.